packages/core/README.md

```diff
@@ -0,0 +1,115 @@
+	</p>
+	<p>
+		<a href="https://vercel.com/?utm_source=discordjs&utm_campaign=oss"><img src="https://raw.githubusercontent.com/discordjs/discord.js/main/.github/powered-by-vercel.svg" alt="Vercel" /></a>
+	</p>
+</div>
+
+## About
+
+`@discordjs/core` is a thinly abstracted wrapper around the "core" components of the Discord API: REST, and gateway.
+
+## Installation
+
+**Node.js 16.9.0 or newer is required.**
+
+```sh-session
+npm install @discordjs/core
+yarn add @discordjs/core
+pnpm add @discordjs/core
+```
+
+## Example usage
+
+```ts
+import { REST } from '@discordjs/rest';
+import { WebSocketManager } from '@discordjs/ws';
+import { Gateway, GatewayIntentBits, InteractionType, MessageFlags, createClient } from '@discordjs/core';
+
+// Create REST and WebSocket managers directly
+const rest = new REST({ version: '10' }).setToken(token);
+const ws = new WebSocketManager({
+	token,
+	intents: GatewayIntentBits.GuildMessages | GatewayIntentBits.MessageContent,
+	rest,
+});
+
+// Create a client to emit relevant events.
+const client = createClient({ rest, ws });
+
+// Listen for interactions
+// Each event contains an `api` prop along with the event data that allows you to interface with the Discord REST API
+client.on('interactionCreate', async ({ interaction, api }) => {
+	if (!(interaction.type === InteractionType.ApplicationCommand) || interaction.data.name !== 'ping') {
+		return;
+	}
+
+	api.interactions.reply(interaction.id, interaction.token, { content: 'Pong!', flags: MessageFlags.Ephemeral });
+});
+
+// Listen for the ready event
+client.on('ready', () => console.log('Ready!'));
+
+// Start the WebSocket connection.
+ws.connect();
+```
+
+## Independent REST API Usage
+
+```ts
+// Create REST instance
+const rest = new REST({ version: '10' }).setToken(token);
+
+// Pass into API
+const api = new API(rest);
+
+// Fetch a guild using the API wrapper
+const guild = await api.guilds.get('1234567891011');
+```
+
+## Links
+
+- [Website][website] ([source][website-source])
+- [Documentation][documentation]
+- [Guide][guide] ([source][guide-source])
+  See also the [Update Guide][guide-update], including updated and removed items in the library.
+- [discord.js Discord server][discord]
+- [Discord API Discord server][discord-api]
+- [GitHub][source]
+- [npm][npm]
+- [Related libraries][related-libs]
+
+## Contributing
+
+Before creating an issue, please ensure that it hasn't already been reported/suggested, and double-check the
+[documentation][documentation].  
+See [the contribution guide][contributing] if you'd like to submit a PR.
+
+## Help
+
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
+nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+
+[website]: https://discord.js.org/
+[website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
+[documentation]: https://discord.js.org/#/docs/core
```

Comment:
If the website is https://discord.js.org/, then `[website-source]` isn't correct. Should be the website repository, or modify the documentation and website link?